docker/env.example

```diff
@@ -1,6 +1,11 @@
 # TODO: Replace with your RPC URL
-export RPC_URL=https://holesky.drpc.org
+RPC_URL=https://holesky.drpc.org
 
-export CHAIN_ID=17000
-export ETHERSCAN_API_KEY=foobar
-export LOCAL_ETHEREUM_RPC_URL="http://localhost:8545"
\ No newline at end of file
+DEPLOY_ENV=TESTNET
+CHAIN_ID=17000
+ETHERSCAN_API_KEY=foobar
+LOCAL_ETHEREUM_RPC_URL=http://localhost:8545
+
+# what should I put here??
```

Comment:
@RonTuretzky I need to fill these in for it to run, but what are proper default values?

I grabbed some strategy address and used it for both